addon/templates/components/uni-progress-bar.hbs

```diff
@@ -1 +1,5 @@
 <div role="progressbar" class="uni-progress-bar__bar" style={{inlineStyle}}></div>
+
+{{#x-times totalStepsCount as |step|}}
```

Comment:
This can be done without installing an addon as we're already doing that on AP.

Create an util that creates an array given a range. You copy [this one](https://github.com/uniplaces/uniplaces-ap/blob/master/web/assets/emberapp/app/utils/array-methods.js).

Then, just iterate over the array:

```hbs
{{#each totalStepsCountAsArray}}
{{/each}}
```